src/api/object_api.js

```diff
@@ -47,7 +47,13 @@ module.exports = {
                     },
                     lock_settings: {
                         $ref: 'common_api#/definitions/lock_settings'
-                    }
+                    },
+                    etag: {
+                        type: 'string'
+                    },
+                    partial_object: {
```

Comment:
remove `partial_object`?

Reply:
We need the input parameter. See how it is used at https://github.com/noobaa/noobaa-core/blob/e4c92b74a59f5ba873ed28737989d3ef53893fc5/src/server/object_services/object_server.js#L75

Reply:
Got it. How about we rename it to something like `complete_upload` so that it will essentially make the object "completed" and not require another call to `complete_object_upload`.